src/components/feedbackpage/feedbackpage.scss

```diff
@@ -0,0 +1,38 @@
+#line-1, #line-4{
+    font-style: italic;
+}
+#line-2,#line-3{
```

Comment:
Add a blank line after the end of a CSS block. Also add a space after every punctuation like a comma and special characters like `{`